File: src/main/java/fi/metatavu/metaform/server/pdf/PdfRenderException.java
package fi.metatavu.metaform.server.pdf;

/**
 * Exception throw when PDF generation fails
 * 
 * @author Antti Leppä
 */
public class PdfRenderException extends Exception {

  private static final long serialVersionUID = -835234745158422224L;

  public PdfRenderException(String message, Throwable original) {
    super(message, original);
  }
  
  public PdfRenderException(String message) {
    super(message);
  }
  
}

File: src/main/kotlin/fi/metatavu/metaform/server/exceptions/PdfRenderException.kt
package fi.metatavu.metaform.server.exceptions

/**
 * Exception throw when PDF generation fails
 *
 * @author Antti Leppä
 */
class PdfRenderException: Exception {
    constructor(message: String?, original: Throwable?) : super(message, original)

    constructor(message: String?) : super(message)

    companion object {
        private const val serialVersionUID = -835234745158422224L
    }
}